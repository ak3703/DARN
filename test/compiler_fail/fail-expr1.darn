int a;
bool b;

void foo(int c, bool d){
  a = 2;
  a * 3;
  d + a; /* Error: bool + int */
}

int main() {
  return 0;
}
