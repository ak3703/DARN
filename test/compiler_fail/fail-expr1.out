Fatal error: exception Failure("Illegal binary operator bool + int in d + a")
